css/jquery.uls.grid.css

```diff
@@ -50,30 +53,25 @@
 	margin: 0 -5px;
 }
 
+.grid .column,
+.grid .columns {
+	float: left;
+	min-height: 1px;
+	padding: 0 5px;
+	position: relative;
+}
+
 .grid .row.collapse .column,
 .grid .row.collapse .columns {
 	padding: 0;
 }
 
-.grid .row .row {
```

Comment:
Removal of this block looks scary. A mistake?

Reply:
Duplicate of -.grid .row .row above

Reply:
Ah okay, I didn't notice it before.